chains/inputs/bad_per_user_key_reverse_sig.iced

```diff
@@ -0,0 +1,21 @@
+chain :
+  user : "max32"
+  ctime : "now"
+  expire : 10000000
+  links : [
+    {
+      type : "eldest"
+      label : "e"
+      key : gen : "eddsa"
+    },
+    {
+      ctime : "+100"
+      label : "puk1"
+      type : "per_user_key"
+      signer : "e"
+      corrupt_v1_proof_hooks :
+        corrupt : ({obj}) ->
+          obj.body.per_user_key.reverse_sig = 'a' + obj.body.per_user_key.reverse_sig[1..]
```

Comment:
instead of corrupting the sig, can you corrupt the thing the sig is computed over?  that would be more interesting.